music_assistant/common/models/config_entries.py

```diff
@@ -365,9 +365,9 @@ class CoreConfig(Config):
     key=CONF_VOLUME_NORMALIZATION_TARGET,
     type=ConfigEntryType.INTEGER,
     range=(-30, 0),
-    default_value=-14,
+    default_value=-17,
     label="Target level for volume normalization",
-    description="Adjust average (perceived) loudness to this target level, " "default is -14 LUFS",
+    description="Adjust average (perceived) loudness to this target level, " "default is -17 LUFS" "WARNING: Setting levels higher than this may result in clipping",
```

Comment:
```suggestion
    description="Adjust average (perceived) loudness to this target level, "
       "default is -17 LUFS" "WARNING: Setting levels higher than this may result in clipping",
```